README.md

```diff
@@ -23,10 +23,19 @@ agent is NOT backwards-compatible with a config file from 0.1.4 and below.
 That being said, the difference is not huge, see below for an example on
 how to setup the new config file.
 
-```
-http://get.influxdb.org/telegraf/telegraf_0.1.5_amd64.deb
-http://get.influxdb.org/telegraf/telegraf-0.1.5-1.x86_64.rpm
-```
+As well, due to a breaking change to the InfluxDB integer line-protocol, there
+are some InfluxDB compatibility requirements:
+
+* InfluxDB <= 0.9.2 requires Telegraf <= 0.1.4
+* InfluxDB > 0.9.2 requires Telegraf > 0.1.4 (this includes any nightly builds)
```

Comment:
I would invert these, and convert them to our version styles (very new addition, not even remotely consistent yet)

InfluxDB 0.9.3+ (including nightly builds) requires Telegraf 0.1.5+ 
InfluxDB 0.9.2 and prior requires Telegraf 0.1.4 

I don't think we need to reference earlier versions of Telegraf. Is there a reason to use anything but 0.1.4 with InfluxDB 0.9.0 through 0.9.2?